qays/style.css

```diff
@@ -0,0 +1,274 @@
+*{
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+
+}
+
+header{
+    flex-direction: row;
+    margin: 16px;
+
+}
+header nav{
+    display: flex;
+    justify-content: space-between;
+}
+
+
+img[alt="logo"]{
+    height: 80px ;
+}
+nav ul{
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+}
+nav ul li a{
+ margin-left: 10px; 
+ text-decoration: none;        
+}
+
+.column-about{
+order: 4;                    
+}
+
+.column-skills{
+    order: 2;                    
+}
+.column-projects{
+    order: 1;                    
+}
+
+.column-contact-me{
+    order: 3;                    
+}
+
+.intro{
+  height:50vh;
+  display:flex;
+  align-items:center;
+  flex-direction: column;
+  justify-content: center;
+
+}
+.about-me-section{
+    display:flex;
+    align-items:center;
+    flex-direction: column;
+    justify-content: center;
+height:50vh;
+background-color:lightgray;
+padding:10%;
+}
+.about-me{
+    padding: 5%;
+    width: 70vw;
+  align-items:center;
+  flex-direction: column;
+  justify-content: center;
+ background-color:darkgray;
+}
+h2,h3{
+    text-align:center;
+
+}
+
+.my-skills-section {
+   height:60vh;
+ 
+    
+
+}
+
+.my-skills-section > div {
+    height:100%;
+  
+     
+ 
+ }
+.my-skills-section [alt="icon"]{
```

Comment:
I wouldn't really use the [alt="X"] much since you can trigger the image without it